geocoding_platform_interface/lib/src/models/location.dart

```diff
@@ -61,8 +61,8 @@ class Location {
         : null;
 
     return Location._(
-      latitude: locationMap['latitude'] ?? null,
-      longitude: locationMap['longitude'] ?? null,
+      latitude: locationMap['latitude'],
+      longitude: locationMap['longitude'],
```

Comment:
Let's add some checks to check for `null` values when parsing the `locationMap` and throw an argument error. Since this should really not happen.